stages/org.osbuild.coreos.live-artifacts.mono

```diff
@@ -584,43 +584,40 @@ def mkrootfs_metal(paths, workdir, img_metal, fstype, fsoptions, loop_client):
             # Make sure to create it, if it is not created yet.
 
             # Remove the sysroot=readonly flag, see https://github.com/ostreedev/ostree/issues/1921
-            subprocess.check_call(['sed', '-i', '/readonly=true/d', f'{tmp_squashfs_dir}/ostree/repo/config'])
+            subprocess.check_call(['sed', '-i', '/readonly=true/d', f'{tmp_rootfs_dir}/ostree/repo/config'])
 
             # And ensure that the kernel binary and hmac file is in the place that dracut
             # expects it to be; xref https://issues.redhat.com/browse/OCPBUGS-15843
 
-            kernel_binary = glob.glob(f"{tmp_squashfs_dir}/boot/ostree/*/vmlinuz*")[0]
-            kernel_hmac = glob.glob(f"{tmp_squashfs_dir}/boot/ostree/*/.*.hmac")[0]
+            kernel_binary = glob.glob(f"{tmp_rootfs_dir}/boot/ostree/*/vmlinuz*")[0]
+            kernel_hmac = glob.glob(f"{tmp_rootfs_dir}/boot/ostree/*/.*.hmac")[0]
             kernel_binary_basename = os.path.basename(kernel_binary)
             kernel_hmac_basename = os.path.basename(kernel_hmac)
 
             # Create hard links in the /boot directory
-            os.link(kernel_hmac, f"{tmp_squashfs_dir}/boot/{kernel_hmac_basename}")
-            os.link(kernel_binary, f"{tmp_squashfs_dir}/boot/{kernel_binary_basename}")
+            os.link(kernel_hmac, f"{tmp_rootfs_dir}/boot/{kernel_hmac_basename}")
+            os.link(kernel_binary, f"{tmp_rootfs_dir}/boot/{kernel_binary_basename}")
 
-            print(f"Kernel binary linked: {tmp_squashfs_dir}/boot/{kernel_binary_basename}")
-            print(f"Kernel HMAC linked: {tmp_squashfs_dir}/boot/{kernel_hmac_basename}")
-            # Generate root squashfs
+            print(f"Kernel binary linked: {tmp_rootfs_dir}/boot/{kernel_binary_basename}")
+            print(f"Kernel HMAC linked: {tmp_rootfs_dir}/boot/{kernel_hmac_basename}")
+            # Generate root squashfs/erofs
             print(f'Creating {fstype} with {fsoptions}')
 
-            # Note the filename must be exactly "root.squashfs" because the 20live
-            # dracut module makes assumptions about the length of the name in sysroot.mount
-            # this matches the set of flags we implicitly passed when doing this
-            # through libguestfs' mksquashfs command
+            # Note the filename must be exactly "root.[squash|ero]fs" because the 20live
```

Comment:
```suggestion
            # Note the filename must be exactly "root.[squash|ero]fs"
            # because the 20live dracut module requires it.
```